src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/item.scss

```diff
@@ -2,18 +2,34 @@
 
 $itemBackgroundColor: $alabaster;
 $itemTextColor: $blueZodiac;
+$itemTextActiveColor: $shakespeare;
 $itemTextColorDisabled: $silver;
+$itemActiveBackgroundColor: $wildSand;
 
 $itemBorder: 1px $wildSand solid;
 
+.container {
```

Comment:
The name container confused me a bit... I don't think it is a good idea to repeat the same names in two sibling files...